libs/utils/Logger.py

```diff
@@ -0,0 +1,179 @@
+import logging
+from asyncio import Queue
+from tabulate import tabulate
+
+
+class Logger:
+    def __init__(self):
+        self.logger = logging.getLogger('log')
+        self.logger.setLevel('INFO')
+
+        filehandler_dbg = logging.FileHandler('MSPannotator.log', mode='w')
```

Comment:
The filename could contain a timestamp in order to be unique so that it is clear which log comes from which run.

Reply:
ideally this would log to a file specified as an argument

Reply:
In 7f3dd17 added option to specify log filename, default contains current datetime.